<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FisheyePanel</class>
 <widget class="QWidget" name="FisheyePanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>207</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolButton
{
border: 0px;
padding: 0px;
}
QToolButton:hover
{
background-color: rgba(44, 122, 158, 220);
}
QToolButton:disabled
{
background: transparent;
}

QLabel:disabled
{
background: transparent;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget_translucent" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="MyLabel" name="label_enable">
        <property name="text">
         <string>Client-side Dewarping</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="MyIconToolButton" name="toolButton_enable">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../resource/resource.qrc">
            <normaloff>:/fisheye2/fisheye2/fisheye_client-side_dewarping_white.png</normaloff>:/fisheye2/fisheye2/fisheye_client-side_dewarping_white.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <property name="maximumSize">
           <size>
            <width>37</width>
            <height>37</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="maximumSize">
           <size>
            <width>37</width>
            <height>37</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="widget_mode" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="MyLabel" name="label_installMode">
           <property name="text">
            <string>Installation Mode</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="MyIconToolButton" name="toolButton_install_ceiling">
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="MyIconToolButton" name="toolButton_install_wall">
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="MyIconToolButton" name="toolButton_install_flat">
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="MyLabel" name="label_displayMode">
           <property name="text">
            <string>Display Mode</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_display">
           <item row="2" column="0">
            <widget class="MyIconToolButton" name="toolButton_1P4R">
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="MyIconToolButton" name="toolButton_1O">
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="MyIconToolButton" name="toolButton_1P1R">
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="MyIconToolButton" name="toolButton_1P">
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="MyIconToolButton" name="toolButton_4R">
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="MyIconToolButton" name="toolButton_1O8R">
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="MyIconToolButton" name="toolButton_1P6R">
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="MyIconToolButton" name="toolButton_1O3R">
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="MyIconToolButton" name="toolButton_2P">
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLabel</class>
   <extends>QLabel</extends>
   <header>mylabel.h</header>
  </customwidget>
  <customwidget>
   <class>MyIconToolButton</class>
   <extends>QToolButton</extends>
   <header>myicontoolbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>toolButton_enable</tabstop>
  <tabstop>toolButton_install_ceiling</tabstop>
  <tabstop>toolButton_install_wall</tabstop>
  <tabstop>toolButton_install_flat</tabstop>
  <tabstop>toolButton_1O</tabstop>
  <tabstop>toolButton_1P</tabstop>
  <tabstop>toolButton_2P</tabstop>
  <tabstop>toolButton_4R</tabstop>
  <tabstop>toolButton_1O3R</tabstop>
  <tabstop>toolButton_1P1R</tabstop>
  <tabstop>toolButton_1P4R</tabstop>
  <tabstop>toolButton_1P6R</tabstop>
  <tabstop>toolButton_1O8R</tabstop>
 </tabstops>
 <resources>
  <include location="../resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
